src/pamela/htn.clj

```diff
@@ -2622,7 +2622,7 @@
   (tpn/reinitialize-tpn-plan-map)
   (reinitialize-pclass-instances)
   (let [[pclass pargs method mi args pca] (identify-root-task ir root-task)
-        _ (dbg-println :info "PAMIA" pclass pargs "METHOD" method mi
+        _ (dbg-println :info "PAMELA" pclass pargs "METHOD" method mi
```

Comment:
Let's revert this back to "PAMIA".
The convention for most `dbg-println` calls is that first string arg is a label (usually an acronym) that identifies where we are in the call stack and/or what follows in the debug output.